### DISCUSSION TITLE
MissingMethodException when trying to declare a resnet18 model

### DISCUSSION THREAD

**Discussion by MLSMITH-WSS** [xamarin/TorchSharp#1184] [Q&A]
Hello! Wanted to see if anyone would be able to provide some guidance with using pre-built torchsharp models like resnet18. I am trying to declare it with the following line, but it's throwing a MissingMethodException when I do so. Stating the Module.load method is not found. Here's the exact exception text:

System.MissingMethodException: 'Method not found: 'Module Module.load(System.String, Boolean, System.Collections.Generic.IList`1<System.String>)'.'

I am using the TorchSharp 0.101.1 package and TorchVision 0.100.7 and I have the libtorch-cpu-win-x64 as well.

**Comment by NiklasGustafsson** [comment]
See my reply to your issue.
